test/modules/ModuleLib/TestResults/DeploymentTestResult.cs

```diff
@@ -3,19 +3,20 @@ namespace Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults
 {
     using System;
     using System.Collections.Generic;
+    using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Azure.Devices.Edge.Util.Json;
 
     public class DeploymentTestResult : TestResultBase
     {
-        public DeploymentTestResult(string source, DateTime createdAt)
+        public DeploymentTestResult(string trackingId, string source, Dictionary<string, string> environmentVariables, DateTime createdAt)
```

Comment:
`trackingId` is passed in but not used?

Reply:
it is used when calling GetFormattedResult() which will cast this class to json.

Reply:
`trackingId ` is not set in the constructor

Reply:
good catch